src/main/kotlin/org/jetbrains/research/ml/ast/transformations/ifRedundantLinesRemoval/IfRedundantLinesRemover.kt

```diff
@@ -0,0 +1,290 @@
+package org.jetbrains.research.ml.ast.transformations.ifRedundantLinesRemoval
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.siblings
+import com.jetbrains.python.psi.*
+import org.jetbrains.research.ml.ast.transformations.PerformedCommandStorage
+import org.jetbrains.research.ml.ast.transformations.PyUtils
+import org.jetbrains.research.ml.ast.transformations.constantfolding.PyEvaluatorImproved
+import org.jetbrains.research.ml.ast.transformations.createBinaryOperandList
+import org.jetbrains.research.ml.ast.transformations.createExpressionStatement
+import org.jetbrains.research.ml.ast.transformations.createIfPart
+import org.jetbrains.research.ml.ast.transformations.createIfPartFromIfPart
+import org.jetbrains.research.ml.ast.transformations.createPrefixExpression
+import org.jetbrains.research.ml.ast.transformations.safePerformCommand
+import org.jetbrains.research.ml.ast.transformations.safePerformCommandWithResult
+import kotlin.test.fail
+
+class IfRedundantLinesRemover(
```

Comment:
This class is so big, can we split it somehow? For example, extract all functions about common suffix and prefix? Or anything else?

Reply:
Sorry, don't see any changes, so what do you think about splitting the class?

Reply:
I made two inner classes `StatementRemover` and `StatementSimplifier` to separate some logic